activerecord/test/cases/relations_test.rb

```diff
@@ -207,14 +207,8 @@ def test_pluck_with_quoted_original_table_name_in_from
   end
 
   def test_select_with_subquery_in_from_uses_original_table_name
-    if current_adapter?(:SQLite3Adapter) && ENV["CI"]
-      skip <<~MSG
-        https://travis-ci.org/rails/rails/jobs/496726410#L1198-L1208
-        https://buildkite.com/rails/rails/builds/58981#2423c707-7c56-4639-a76e-8db4fd1e5cf3/102-111
-      MSG
-    end
     relation = Comment.joins(:post).select(:id).order(:id)
-    subquery = Comment.from(Comment.all, Comment.quoted_table_name).joins(:post).select(:id).order(:id)
+    subquery = Comment.from(Comment.all.distinct, Comment.quoted_table_name).joins(:post).select(:id).order(:id)
```

Comment:
Can you note about that like here? We can drop the workaround if older SQLite3 support is dropped in the future.

https://github.com/rails/rails/blob/a796de1bd644b54e2c27577319c262af0de56e8f/activerecord/lib/arel/visitors/mysql.rb#L68-L73

Reply:
:+1: Make sense. I fixed.